config_parser.cc

```diff
@@ -152,6 +152,7 @@ bool NginxConfigParser::Parse(std::istream* config_file, NginxConfig* config) {
   config_stack.push(config);
   TokenType last_token_type = TOKEN_TYPE_START;
   TokenType token_type;
+  int counter = 0;
```

Comment:
Consider maybe using a more descriptive name... counter could refer to a lot of things so it might be useful to indicate what you're counting!